packages/modal/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "pcln-modal",
+  "version": "1.0.0",
+  "description": "React component for modal",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "jest",
+    "prepare": "rollup --config",
+    "clean": "rm -rf node_modules dist"
+  },
+  "author": "Priceline",
+  "license": "MIT",
+  "dependencies": {
+    "@reach/dialog": "^0.1.2",
+    "react-transition-group": "^2.5.3"
+  },
+  "peerDependencies": {
+    "pcln-design-system": "^1.0.1",
```

Comment:
should this be `>=1.0.1 . || >=2.0.1` ?

Reply:
This thing actually has a lot of dependencies from DS, it actually works with both, but open to discussion on this

Reply:
Definitely should be `>=1.0.0 || >=2.0.0` in my opinion.  If anyone uses pcln-design-system v2 in their app, I think they would see a warning that they are not using the correct version, when in fact, there's no issue.

Reply:
will update this one